GOVERNANCE.md

```diff
@@ -194,11 +193,16 @@ Voting for first election is restricted to establish stable governance, and to d
 ### Core Contributors
 Core Contributors are those who have provided consistent and meaningful contributions to ArviZ.
 These can be, but are not limited to, code contributions, community contributions, tutorial
-development etc.
+development etc. Core Contributors will be given the ability to manage the ArviZ github
+repository, including code merges to master. This does not necessarily mean Core Contributors
+must submit code, but more so signifies trust with the project as a whole.
+
+#### Core Contributor Responsibilities
+* Enforce code of conduct
+* Maintain a check against Council
 
 #### Core Contributor Nominations and Confirmation Process
-Current Core Contributors can nominate candidates for consideration by the council. The council
-can make the determination for acceptance with a process of their choosing.
+Current Core Contributors can nominate candidates to become Core Contributors. If nominated candidates accept their nomination (explicit comment approving nomination on the GitHub issue or "thumbsup" emoji on the same issue), then they can be considered by the Council: on the first of the month following a nomination, the Council will vote on each nominee, where greater than 50% of the votes means a confirmation. In the event of a tie, a second election will occur, where all core contributors can vote. If 50% or greater of the community confirm, the nominee will be accepted.
```

Comment:
The second sentence assumes nominations are public over GitHub issues if I am correctly understanding the proposal but it is not clear nor explicit. Would we open an issue per contributor to be added? Or would there be a core contributor nomination issue where we'd add new comments every time someone is nominated? 

Regarding the voting results, how public should it be? Do council members vote publicly or privately? Is the whole result published or only yes/no (probably better not yet than no)?

Reply:
Should also be updated after deciding the voting style

Reply:
You're right -- Didn't realize this was for Core Contributors. Maybe easier and more practical to do all that in Slack (we actually talked about that before and agreed on Slack 😅)? And we can decide on a yes-no-neutral basis with the same rules?

Reply:
I think yes no neutral with same rules is good. I do think the counts of votes for at least proposals should be visible publicly and not just in slack. I'm not sure if each person individual vote should be public though